src/pages/pmarket/pmarket.css

```diff
@@ -0,0 +1,10 @@
+.tsc-company-spy {
+  display: inline;
+  background-color: var(--tsc-bg-color);
+  color: var(--tsc-text-color);
+  border: 1px solid var(--tsc-border-color);
+  border-radius: 5px;
+  padding: 3px 5px;
+  cursor: pointer;
+  margin-left: 5px;
+}
```

Comment:
You could either completely remove this file (since the company css is already present by the time the points market function runs) or change the class name.